examples/healthcareapis/healthcaremedtechservicefhirdestination.yaml

```
@@ -0,0 +1,247 @@
+
+---
+
+apiVersion: healthcareapis.azure.upbound.io/v1beta1
+kind: HealthcareMedtechService
+metadata:
+  annotations:
+    meta.upbound.io/example-id: healthcareapis/v1beta1/healthcaremedtechservicefhirdestination
+  labels:
+    testing.upbound.io/example-name: example
+  name: hmsfdex
+spec:
+  forProvider:
+    deviceMappingJson: |
+      {
+          "templateType": "CollectionContent",
+          "template": [
+                      {
+                        "templateType": "JsonPathContent",
+                        "template": {
+                          "typeName": "heartrate",
+                          "typeMatchExpression": "$..[?(@heartrate)]",
+                          "deviceIdExpression": "$.deviceid",
+                          "timestampExpression": "$.measurementdatetime",
+                          "values": [
+                            {
+                              "required": "true",
+                              "valueExpression": "$.heartrate",
+                              "valueName": "hr"
+                            }
+                          ]
+                        }
+                      }
+                    ]
+      }
+    eventhubConsumerGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    eventhubNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    eventhubNamespaceNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    identity:
+    - type: SystemAssigned
+    location: east us
+    workspaceIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: eventhub.azure.upbound.io/v1beta1
+kind: EventHub
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kusto/v1beta1/healthcaremedtechservicefhirdestination
+  labels:
+    testing.upbound.io/example-name: example
+  name: hmsfdex
+spec:
+  forProvider:
+    messageRetention: 1
+    namespaceNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    partitionCount: 1
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: eventhub.azure.upbound.io/v1beta1
+kind: ConsumerGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kusto/v1beta1/healthcaremedtechservicefhirdestination
+  labels:
+    testing.upbound.io/example-name: example
+  name: hmsfdex
+spec:
+  forProvider:
+    eventhubNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    namespaceNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: eventhub.azure.upbound.io/v1beta1
+kind: EventHubNamespace
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kusto/v1beta1/healthcaremedtechservicefhirdestination
```

```suggestion
    meta.upbound.io/example-id: healthcareapis/v1beta1/healthcaremedtechservicefhirdestination
```
could you please change the group name?